playbooks/_buildapp.yml

```diff
@@ -0,0 +1,20 @@
+---
+
+- name: Build Application
+  hosts: fpm[0]
```

Comment:
We're assuming here that this is a PHP app, which for now is 100% fine.  Just need to keep in the back of our heads that this playbook will ultimately be lang agnostic.